unbound/metadata.csv

```diff
@@ -0,0 +1,41 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+unbound.thread0.num.queries,rate,,,,number of queries received by thread,1,unbound,thread0.num.queries
```

Comment:
If this is supposed to list every possible metric this could generate, I'm in a bit of a pickle....or maybe I need to structure the code differently.  The number of threads is configurable, and could be big, so the way the code is currently written we could have unbound.thread0.*, unbound.thread1.*, unbound.thread2.*, etc.  I suppose this means I need to pick off the thread number and use it as a tag...How do other folks handle this?

Reply:
Gave it a shot in https://github.com/DataDog/integrations-extras/pull/451/commits/ece0b148f971b032da36c4c3b11d516a314b2c20

Reply:
> I suppose this means I need to pick off the thread number and use it as a tag.

Yes, using tags for this case seems like a good idea.

Reply:
Found a few more cases like this that I still need to address.

Reply:
Done with this.